cmd/amp/commands.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"github.com/appcelerator/amp/cli"
+	"github.com/appcelerator/amp/cli/command/version"
+	"github.com/spf13/cobra"
+)
+
+type Opts struct {
```

Comment:
Thanks. I just started doing the lint checks and can also see the CI build results. I still need to clean up the PR, squash, and won't expect this merged until the CI build passes.